tests/components/grid.test.ts

```diff
@@ -639,6 +642,46 @@ describe("Grid component", () => {
     });
   });
 
+  describe("filter icon vertical alignment", () => {
+    let centeringOffset: number;
```

Comment:
centeringOffset is only used once, no need to put it in the beforeEAch

Reply:
@hokolomopo implemented requested changes.please check